components/gcp/container/component_sdk/python/kfp_component/google/bigquery/_query.py

```diff
@@ -79,6 +113,29 @@ def cancel():
         _dump_outputs(query_job, output_gcs_path, table_ref)
         return query_job.to_api_repr()
 
+def create_table_from():
```

Comment:
Perhaps add a docstring briefing the purpose of this function?

Also it seems like not being used at this moment.